src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue32287.cs

```diff
@@ -0,0 +1,20 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue32287 : _IssuesUITest
+{
+	public Issue32287(TestDevice testDevice) : base(testDevice) { }
+
+	public override string Issue => "Using custom TitleView in AppShell causes shell content to be covered in iOS 26";
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void CustomTitleViewDoesNotCoverContent()
+	{
+		App.WaitForElement("Label");
+		VerifyScreenshot();
```

Comment:
Pending snapshots already available in the latest build:

<img width="1080" height="1735" alt="Image" src="https://github.com/user-attachments/assets/097fefee-8239-41c1-b752-cb2e822df07f" />

Could you commit the images?